src/Resources/skeleton/test/Functional.tpl.php

```diff
@@ -12,6 +12,6 @@ public function testSomething()
         $crawler = $client->request('GET', '/');
 
         $this->assertSame(200, $client->getResponse()->getStatusCode());
-        $this->assertSame(1, $crawler->filter('h1:contains("Hello World")')->count());
+        $this->assertCount(1, $crawler->filterXPath('//h1:[contains(text(), "Hello World")]'));
```

Comment:
What about
```php
$this->assertContains('Hello World', $crawler->filter('h1'));
```

Reply:
should be this though:

```php
$this->assertContains('Hello World', $crawler->filter('h1')->text());
```

And this assumes that it is the only h1 on the page btw (or at least the first one), otherwise the test would fail

Reply:
You are right ;)

But anyway, this is boiler plate code, and it should be updated...
So I'm totally :+1: with stof's proposal